invoice/src/test/java/org/killbill/billing/invoice/dao/MockInvoiceDao.java

```diff
@@ -134,6 +134,13 @@ public InvoiceModelDao getById(final UUID id, final InternalTenantContext contex
         }
     }
 
+    @Override //TODO_1951 - revisit to check if anything else needs to be done
```

Comment:
It's fine, you can remove TODO - if not of the tests break, this mock does not use `is_repair` functionality. If later we need it, we can add it.